languages.php

```diff
@@ -104,6 +104,13 @@
                                 Rust-SDL2 -
                                 <a href="https://github.com/AngryLawyer/rust-sdl2">https://github.com/AngryLawyer/rust-sdl2</a>
                             </li>
+                            <li> <strong>
+                                    Haskell

```

Comment:
The language should be added in alphabetical order.

Reply:
Done